apps/l10nstats/management/commands/progress.py

```diff
@@ -121,6 +124,25 @@ def rescale(self, vals):
         scale = lambda v: (v - _min) * ratio + bottom
         return [(t, scale(c)) for t, c in vals]
 
+    def initialCoverage(self, tuples, startdate):
+        rv = {}
+        tree2locs = defaultdict(list)
+        for l,t in tuples:
+            tree2locs[t].append(l)
+        for tree, locales in tree2locs.iteritems():
+            locs = (Locale.objects
+                .filter(code__in=locales)
+                .filter(run__srctime__lt=startdate, run__tree__code=tree)
```

Comment:
Hm, for readability sake the query should use the reverse name of the query, which is `run_set` since you haven't specified a `related_name` parameter in the `ForeignKey`. In 1.6 there was a new paramater added called `related_query_name` where you can specify the name of the query parameter independent from the name of the related attribute (https://github.com/django/django/commit/99b467f272da91b8894dc90d793d8d2c40b78d8c). Your query happens to work since your model is called `Run` and therefore the fallback of using the lower-cased model named works.